app/views/providers/check_provider_answers/index.html.erb

```diff
@@ -1,11 +1,11 @@
 <%= page_template(
-      page_title: t('.title'),
+      page_title: @read_only ? '' : t('.title'),
       back_link: {
-        path: reset_providers_legal_aid_application_check_provider_answers_path,
-        method: :post
+        path: @read_only ? providers_legal_aid_applications_path : reset_providers_legal_aid_application_check_provider_answers_path,
+        text: @read_only ? t('generic.home') : t('generic.back'),
+        method: @read_only ? :get : :post
```

Comment:
I think this would be clearer:
```erb
<%
  read_only_back_link = {
      path: providers_legal_aid_applications_path,
      text: t('generic.home'),
      method: :get
    }
  back_link = {
      path: reset_providers_legal_aid_application_check_provider_answers_path,
      text: t('generic.back'),
      method: :post
    }  
%>
<%= page_template(
      page_title: @read_only ? '' : t('.title'),
      back_link: @read_only ? read_only_back_link : back_link
  ) %>

Reply:
That makes a lot more sense, thanks. 👍